src/common/components/Currency/CurrencyValue.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue } from '@folio/stripes/components';
+
+const CurrencyValue = ({ value }) => (
```

Comment:
I don't see where it's used on Details pane. I suppose currency should be displayed there as well

Reply:
I'm not sure about displaying currency on Datails pane. User can see currency symbols on 'Totals'. [Here](https://issues.folio.org/browse/UINV-187) is the latest mock-up, I guess.